contents/blog/the-posthog-array-1-39-0.mdx

```diff
@@ -0,0 +1,128 @@
+
+Also, it makes queries significantly faster because we no longer have to merge tables to get a result; we can just look up everything in the events table instead. One query in our internal tests showed a 400x increase in speed...but this beta will help us understand the improvement in real-world conditions. 
+
+There'll be some slight changes to how filtering by unique users or person properties work as a result of this change. For more information on this and to find out how we've implemented it change behind the scenes, check out our [overview of the ingestion pipeline](/docs/architecture/ingestion-pipeline).
+
+### New: Format your insights to show durations
+
+![formatting an insight as a duration](../images/blog/array-1.39.0/duration-formatting.gif)
+
+Do your event properties contain milliseconds or seconds? Then you can now make things easier by changing your Y-axis unit to show durations instead of the default `none`. It's a small addition, but one which makes insights much easier to read and share with others.
+
+### New: Big, bold number chart type
+
+![the big bold number insight formatted as a duration](../images/blog/array-1.39.0/big-number.png)
+
+This new, much requested and _very_ exciting big bold number insight type does exactly what it says: simplifies a query result down to a single number. And it works great with durations too!
+
+### Improved: Turbo properties
+
+Frustrated that it was taking a few seconds to load properties? You weren't the only one. 
+
+Thankfully, version 1.39.0 addresses this by adding a limit so that PostHog doesn't always search through hundreds of thousands of properties for each query. The result? [Properties now load much faster](https://github.com/PostHog/posthog/pull/11037). On PostHog Cloud we saw a 75% reduction in API call time. 
+
+### Improved: History log now shows changes to insights
+
+![a view of the insights activity tab](../images/blog/array-1.39.0/insights-activity.png)
+
+We've had feedback from several users that they enjoy tracking changes to their feature flags in the history logs. So, we added the same functionality to insights too! Now you can snoop on who has been changing your insights and giving them funny names.
+
+## Improved: Client and server-side library improvements
+
+While we continue to recommend using Javascript if possible, we've made significant updates to our other client libraries so that Android, iOS, Flutter and React Native now all support feature flags. We've also updated our server-side libraries to have local feature flag evaluation.
+
+Check [our library docs](/docs/integrate/libraries) to find out more about what each library supports and how to deploy. 
+
+### Improved: Automatically comment SQL queries
```

Comment:
I'd remove this, too internal

Reply:
This is worth highlighting under 'other improvements' as clients who we've helped debug things have asked/needed this before.